### PR TITLE
IPU:add stream installation way for leapp tool

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID:2634

Provide installing particular packages for leapp tool

### CODE REVIEW

**Review by PaulYuuu — CHANGES_REQUESTED** [review]
Please make the code flexible with less hard code, and don't provide internal info. cc @lijinlijin

**Comment by MiriamDeng** [comment]
Thanks @PaulYuuu will move the parameter to the internal cfg.

**Comment by MiriamDeng** [comment]
@fbq815 could you also help to review this patch ?

File: qemu/tests/cfg/in_place_upgrade.cfg [hunk 1]
```diff
@@ -53,6 +53,8 @@
         - compose_install:
             com_install = "yes"
             com_ins_leapp = "yum install leapp-upgrade -y"
+        - stream_install:
+            # Please modify the build in the internal cfg
```
Comment (PaulYuuu) [inline qemu/tests/cfg/in_place_upgrade.cfg:57]: I think this should add an example rather than point to internal.

**Comment by MiriamDeng** [comment]
Updated. Can you help to review it ? Thank you !

File: qemu/tests/cfg/in_place_upgrade.cfg (hunk 1 above)
Comment (MiriamDeng) [reply qemu/tests/cfg/in_place_upgrade.cfg:57]: updated

**Comment by fbq815** [comment]
seems there's some issue need to be handle, log is paste in the jira ticket

**Comment by MiriamDeng** [comment]
The issue mentioned by Boqiao, it's a known leapp product issue and I've upgrade related information to ID:2641

**Comment by MiriamDeng** [comment]
Hi @fbq815 ,
I refined the patch and please also apply the following ones at the same time. 
https://github.com/autotest/tp-qemu/pull/4170
https://xxx/1348
Thanks.

**Review by fbq815 — APPROVED** [review]
As the test result and discuss offline, aprove this MR

**Comment by MiriamDeng** [comment]
I've added the "verifie"d in the commit part, thanks.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**